_hoh_floorsets/061.md

```diff
@@ -31,8 +31,9 @@ boss_abilities:
     description: 'telegraphed wide charge AoE to targeted player; inflicts
     stun (4s)'
   - name: Devour
-    description: 'instant death to players WITHOUT minimum debuff (otherwise no
-    damage)'
+    potency: 2500?
+    description: 'instant death to players WITHOUT minimum debuff (otherwise
+    misses); on hit, absorbs target's remaining HP'
```

Comment:
Need double `''``. With this change, it all builds successfully and looks good

Reply:
Whoops, fixed.